src/boilerplate/content.js

```diff
@@ -1,3 +1,5 @@
+import { setScreenVisibility } from "./screens";
```

Comment:
I think at this stage we can do opt-1, like we do in `content.js` already. E.g. in line 94: `document.documentElement.style.setProperty(`--playpass-cms-${key}`, newValue);`.
 
I can add function setScreenVisibility into `content.js` since it is already dealing with DOM objects updates.
LMK what do you think @gabrielmpinto.

Reply:
Sounds good